one/src/migrations.rs

```diff
@@ -93,8 +93,10 @@ async fn from_ipfs(opts: FromIpfsOpts) -> Result<()> {
     let network = opts.network.to_network(&opts.local_network_id)?;
     let db_opts: DBOpts = (&opts).into();
     let sqlite_pool = db_opts.get_sqlite_pool().await?;
+    // the source did:key is none for migrations as we don't know the source node.
```

Comment:
hmm, couldn't an argument be made to just consider this local node the source?  This is the first node bringing this data into the Recon network after all...

Reply:
Maybe, maybe not. It was likely created and anchored by someone else before the migration. e.g. the old node you are migrating from.

Reply:
right, but that's almost certainly "logically" the same node.  Ie operated by the same person, serving the same purpose/application, and directly replacing the old node in every way.  What's the harm in considering the migrated event as having this node as the source?

Reply:
If these events are considered originating locally, they might be considered eligible for self-anchoring.

During migration, it would probably be safe to assume that any events that needed to be anchored were already anchored.